src/nomad_json_parser/__init__.py

```diff
@@ -0,0 +1,97 @@
+from nomad.config.models.plugins import (
+    ParserEntryPoint,
+    SchemaPackageEntryPoint,
+)
+from pydantic import Field
+
+
+class JsonMapperEntryPoint(SchemaPackageEntryPoint):
+    parameter: int = Field(0, description='Custom configuration parameter')
+
+    def load(self):
+        from nomad_json_parser.schema_packages.jsonimport import m_package
+
+        return m_package
+
+
+jsonmapper_entry = JsonMapperEntryPoint(
+    name='JSON Mapper Importer',
+    description='New schema package entry point configuration.',
+)
+
+
+class MappedJsonEntryPoint(SchemaPackageEntryPoint):
+    parameter: int = Field(0, description='Custom configuration parameter')
+
+    def load(self):
+        from nomad_json_parser.schema_packages.jsonimport import m_package
+
+        return m_package
+
+
+mappedjson_entry = MappedJsonEntryPoint(
+    name='JSON Mapped Importer',
+    description='New schema package entry point configuration.',
+)
+
+
+class JsonMapperParserEntryPoint(ParserEntryPoint):
+    def load(self):
+        from nomad_json_parser.parsers.parser import JsonMapperParser
+
+        return JsonMapperParser(**self.dict())
+
+
+jsonmapper_parser = JsonMapperParserEntryPoint(
+    name='MapperParser for Json Mapper files',
+    description="""Parser for Json Mapping files.""",
+    mainfile_name_re=r'.+\.json',
+    mainfile_mime_re='application/json',
+    mainfile_contents_re=r'.+json_mapper_class_key',
+)
+
+
+class MappedJsonParserEntryPoint(ParserEntryPoint):
+    def load(self):
+        from nomad_json_parser.parsers.parser import MappedJsonParser
+
+        return MappedJsonParser(**self.dict())
+
+
+mappedjson_parser = MappedJsonParserEntryPoint(
+    name='JsonParser for Json Mapped files',
+    description="""Parser for Json Mapped files.""",
+    mainfile_name_re=r'.+\.json',
+    mainfile_mime_re='application/json',
+    mainfile_contents_re=r'.+mapped_json_class_key',
+)
+
+
+class MainLevelEntryPoint(SchemaPackageEntryPoint):
```

Comment:
This name is quite confusing. I guess this is the example schema used for the testing and the example upload?
```suggestion
class ExampleSchemaEntryPoint(SchemaPackageEntryPoint):
```